app.js

```diff
@@ -67,6 +67,17 @@ app.post("/import", (req, res) => {
   // TODO write to disk
 });
 
+app.post("/addwallet", async (req, res) => {
+  const { tweet } = req.body;
```

Comment:
should prob be `tweetUrl` to be clear what this endpoint expects.

maybe we should even just pass the tweet id?

Reply:
tweet url was just more convenient for the frontend haha